docs/platform.rst

```diff
@@ -0,0 +1,53 @@
+IDAES Integrated Platform
+=========================
+The IDAES integrated platform (IDAES-IP) supports the full process modeling lifecycle from conceptual design to dynamic optimization and
+control within a single modeling environment. At the center of this platform is the Core Modeling Framework
+(:term:`IDAES-CMF`) which leverages the open source, U.S. Department of Energy-funded extensible algebraic modeling
+environment, :term:`Pyomo`.
+
+Below is a diagram showing the components of the IDAES Integrated Platform (IDAES-IP).
+
+
+.. image:: _images/IDAES-IP.png
+
+This diagram shows the relationship between the major IDAES-IP components:
+
+.. glossary::
+
+    IDAES-IP
+        IDAES integrated platform, described on this page
+
+    IDAES-Core
+        The software package that includes the Core Modeling Framework; process, unit, and property model libraries;
+        data management, artificial intelligence and uncertainty quantification tools; and grapphical user interfaces.
+
+    IDAES-CMF
```

Comment:
I am not sure if we should call this out as a separate part here or not. Notably, it is a part of IDAES-Core (as mentioned above), and we want to be careful we don't confuse people into thinking that this is somehow different from that.

Reply:
One of those people would be me. I thought IDAES-CMF was used to refer to the capability to build the models minus the modeling libraries, and IDAES-Core was the whole thing. The diagram suggests that. Otherwise, IDAES-Core should be labeled IDAES-CMF.

Reply:
You are more-or-less correct with that @dangunter. The IDAES Core Modeling Framework is a large part of IDAES Core (at least at the moment). Note that it might be easier to understand if you read it as "IDAES Core - Modeling Framework".